test-cases/gemini/gemini-3h-ics-cdc-with-nemesis.yaml

```diff
@@ -32,4 +32,3 @@ stress_cdclog_reader_cmd: "cdc-stressor -duration 215m -stream-query-round-durat
 db_type: mixed_scylla
 instance_type_db_oracle: 'i3.8xlarge'
 oracle_scylla_version: "2020.1.7"
-append_scylla_args_oracle: '--enable-cache false'
```

Comment:
I suggest to restore these line in config.Because the oracle cluster use  single node, cache result could affect on result of gemini compare. If nothing changed.

Reply:
this is already what's there in the `test_default.yaml` no point on repeating it again